plugins/defaultplugins/ifplugin/bin_api/stn/stn.go

```diff
@@ -0,0 +1,123 @@
+// Code generated by govpp binapi-generator DO NOT EDIT.
+// Package stn represents the VPP binary API of the 'stn' VPP module.
+// Generated from 'stn.api.json'
+package stn
```

Comment:
please verify that the file is generated against newest version of the VPP (vpp with commit id 4c9f2a805038a2d4f663b05a3d08ac4ee1eec3da), re-generate it if needed